README.md

```diff
@@ -1,70 +1,89 @@
-# Getting Started with Create React App
+# Space Travelers' Hub
+" Space Travelers' Hub is a web application for a company that provides commercial and scientific space travel services. This application allows users to book rockets and join selected space missions."
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+![space](https://user-images.githubusercontent.com/57562869/131994816-227c4d86-0505-4fb6-a7be-ccbb514fec8a.png)
 
-## Available Scripts
 
-In the project directory, you can run:
+## Built With
 
-### `npm start`
+- ReactJS
+- Redux
+- SASS
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+## Live Demo
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+[Live Demo Link](https://AkashaRojee.github.io/space-travelers-hub)
 
-### `npm test`
+## Video
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+[video link](https://www.loom.com/share/48a08a3b0de14dbfa6de0594d31e7dd7)
```

Comment:
- [ ] As stated all members that worked on the project should appear in the video explaining the project. Only 2 members appear. Kindly record a new video with all members.